packages/runtime/src/client/index.ts

```diff
@@ -45,6 +46,8 @@ import { deterministicUUID } from '../utils/deterministic-uuid'
 import { Schema } from '@makeswift/controls'
 import { EMBEDDED_DOCUMENT_TYPE, EmbeddedDocument } from '../state/modules/read-only-documents'
 
+export { SnippetLocation } from '../api/graphql/generated/types'
```

Comment:
Use already existing, GraphQL-derived type definitions for `Snippet`/`SnippetLocation` in place of https://github.com/makeswift/makeswift/pull/1071/files#diff-6fbac26bb292131b8508e29ab56846fec469ade26eae4acd2800f0c566825a1a